runtime/parachains/src/paras.rs

```diff
@@ -52,6 +52,9 @@ pub trait Config: frame_system::Config + configuration::Config {
 		+ Into<result::Result<Origin, <Self as Config>::Origin>>;
 }
 
+// Wait until the session index is 2 larger then the current index to apply any changes.
+const SESSION_DELAY: SessionIndex = 2;
```

Comment:
Can you add some elaboration here or link to somewhere that does elaborate more?

Reply:
targeting this for another PR